cli/main.go

```diff
@@ -340,35 +346,34 @@ func buildDaemonCommandLine(cfg Config, fullManifestPath string) []string {
 	var daemonArgs []string
 	daemonArgs = append(daemonArgs, "-daemon")
 
-	if (*debugFuseFlag) {
+	if *debugFuseFlag {
 		daemonArgs = append(daemonArgs, "-debugFuse")
 	}
-	if (*fsSync) {
+	if *fsSync {
 		daemonArgs = append(daemonArgs, "-sync")
 	}
-	if (*gid != -1) {
-		args := []string { "-gid", strconv.FormatInt(int64(*gid), 10) }
+	if *gid != -1 {
+		args := []string{"-gid", strconv.FormatInt(int64(*gid), 10)}
 		daemonArgs = append(daemonArgs, args...)
 	}
-	if (*readOnly) {
+	if !*readWrite {
 		daemonArgs = append(daemonArgs, "-readOnly")
 	}
```

Comment:
Did we change the interface to dxfuse to use readWrite?
```suggestion
	if readWrite {
		daemonArgs = append(daemonArgs, "-readWrite")
	}
```